newIDE/app/src/EventsSheet/EventsTree/Instruction.js

```diff
@@ -128,6 +130,41 @@ const formatValue = ({
     ? i18n._(t`Base layer`)
     : value;
 
+const InstructionMissing = (props: {|
+  instructionType: string,
+  isCondition: boolean,
+|}) => {
+  const { hasExtensionNamed } = React.useContext(ExtensionStoreContext);
+  const { name, behaviorName, extensionName } = getFunctionNameFromType(
+    props.instructionType
+  );
+  const extensionStoreMention = hasExtensionNamed(extensionName)
+    ? ' Try installing it from the extension store.'
```

Comment:
is the space before the "try" on purpose?
We should probably translate this too! so maybe remove the space here and add it before the mention when you use the mention below.
And use `<Trans>` ?